AntistasiOfficial.Altis/Stringtable.xml

```diff
@@ -708,6 +708,12 @@
 		</Container>
 
 		<Container name="Admin">
+			<Key ID="STR_UI_ADMIN_TGLPN_TEXT">
+				<Original>Map names</Original>
+			</Key>
+			<Key ID="STR_UI_ADMIN_TGLPN_TT">
+				<Original>Toggle player names displayed on map</Original>
```

Comment:
What about `Map marker`s instead of `Map names`

Reply:
Best would be Map player names